tests/test_settings.py

```diff
@@ -15,12 +15,16 @@
     'tests',
 ]
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': ':memory:',
+if 'ENGINE' in base_settings.DATABASES.get('default', {}):
```

Comment:
Can't be necessary to use a spaghetti text hack here? Use a separate subtype of settings for github.

Reply:
Worse is that we're depending so hard on settings in base. I'll pull in "getenv" and redo the code that uses the envvar DMP_DATABASE_URL instead.